src/quotamodel.c

```diff
@@ -188,26 +187,25 @@ struct LocalRejectMapEntry
 };
 
 /* using hash table to support incremental update the table size entry.*/
-static HTAB       *table_size_map                  = NULL;
-static TimestampTz table_size_last_overflow_report = 0;
+static HTAB       *table_size_map                      = NULL;
+static TimestampTz table_size_map_last_overflow_report = 0;
 
 #define TABLE_SIZE_MAP_WARNING                                             \
 	"[diskquota] the number of tables reached the limit, please increase " \
-	"the GUC value for diskquota.max_table_segments. Current "             \
-	"diskquota.max_table_segments value:"
+	"the GUC value for diskquota.max_table_segments."
 
 /* rejectmap for database objects which exceed their quota limit */
 static HTAB *disk_quota_reject_map       = NULL;
 static HTAB *local_disk_quota_reject_map = NULL;
 
-static TimestampTz disk_quota_reject_last_overflow_report       = 0;
-static TimestampTz local_disk_quota_reject_last_overflow_report = 0;
+static TimestampTz disk_quota_reject_map_last_overflow_report       = 0;
+static TimestampTz local_disk_quota_reject_map_last_overflow_report = 0;
 
 #define DISK_QUOTA_REJECT_MAP_WARNING                               \
 	"[diskquota] Shared disk quota reject map size limit reached. " \
-	"Some out-of-limit schemas or roles will be lost in rejectmap. Current limit:"
+	"Some out-of-limit schemas or roles will be lost in rejectmap."
```

Comment:
Maybe add a GUC(s) for these limits, otherwise why issue a warning if nothing can be done about it at all?